apps/events/tests/conftest.py

```diff
@@ -5,28 +5,35 @@
 from typing import List
 
 import pytest
+from django.contrib.gis.geos import Point
 from django.utils import timezone
 
 from apps.events.models import Event, Category, Tag, Currency
+from apps.events.models.city import City
+from apps.profiles.tests.utils import get_tokens
+from rest_framework.test import APIClient
 
 
 @pytest.fixture
-def hundred_events() -> List:
+def hundred_events(city_location_yandex) -> List:
     events = []
     for i in range(100):
         name = ''.join(choices(string.ascii_uppercase + string.digits, k=20))
-
+        city_location = city_location_yandex
         start_date = timezone.now() + datetime.timedelta(days=i)
         end_date = start_date + datetime.timedelta(hours=random.randint(1, 5))
-        start_time = (datetime.datetime.combine(datetime.date.today(), datetime.time()) + datetime.timedelta(minutes=random.randint(1, 1440))).time()
-        end_time = (datetime.datetime.combine(datetime.date.today(), start_time) + datetime.timedelta(hours=random.randint(1, 5))).time()
+        start_time = (datetime.datetime.combine(datetime.date.today(), datetime.time()) + datetime.timedelta(
+            minutes=random.randint(1, 1440))).time()
+        end_time = (datetime.datetime.combine(datetime.date.today(), start_time) + datetime.timedelta(
+            hours=random.randint(1, 5))).time()
 
```

Comment:
better to use specific date values, you have a chance of end time being earlier than start time. Freezegun module helps to set specific dates for your tests so you don't need to mock datetime.now()